util/find-doc-nits

```diff
@@ -151,6 +151,23 @@ sub name_synopsis()
     }
 }
 
+# Check that period ends a line in the text (=head1 DESCRIPTION) block.
+sub check_linebreaks()
+{
+    my $id = shift;
+    my $contents = shift;
+
+    $contents =~ s/=head1 DESCRIPTION(.*)=head1 COPYRIGHT.*/$1/ms;
+    foreach my $line ( split /\n+/, $contents ) {
+        if ( $line !~ /^ /) {
+            # Look for "period space" if not proceeded by i.e. or e.g.
+            print $id, "Linebreak needed:\n  $line\n"
+                if $line =~ /(?<!([Ii]\.e|[Ee]\.g))\. +/;
```

Comment:
I was wondering why not `\s`, but then realised you might not want to match vertical space characters. Tip: `\h` matches horizontal space characters.

Reply:
We don't have tabs and the podpages are ascii, so I think a literal space character is the right thing.

Reply:
Wouldn't a case insensitive regular expression be a bit more readable? `/(?<!(i\.e|e\.g))\. +/i`. 

_(Even if there was an abbreviation like "i.E." (which actually exists in German), the exception would still apply.)_

Reply:
sure, looks more readable, commit pushed.